website/eslint.config.js

```diff
@@ -0,0 +1,146 @@
+import eslint from '@eslint/js';
+import tseslint from 'typescript-eslint';
+import react from 'eslint-plugin-react';
+import parser from '@typescript-eslint/parser';
+import eslintPluginAstro from 'eslint-plugin-astro';
+import astroParser from 'astro-eslint-parser';
```

Comment:
Let's explicitly add `astro-eslint-parser` as a dependency, since we're using it here?